java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java

```diff
@@ -499,10 +507,28 @@ public static ObjectHandle runSecurityAnalysis(IsolateThread thread, ObjectHandl
     }
 
     @CEntryPoint(name = "getSecurityAnalysisResult")
-    public static ArrayPointer<ContingencyResultPointer> getSecurityAnalysisResult(IsolateThread thread, ObjectHandle securityAnalysisResultHandle, ExceptionHandlerPointer exceptionHandlerPtr) {
+    public static ArrayPointer<SeriesPointer> getSecurityAnalysisResult(IsolateThread thread, ObjectHandle securityAnalysisResultHandle, ExceptionHandlerPointer exceptionHandlerPtr) {
         return doCatch(exceptionHandlerPtr, () -> {
             SecurityAnalysisResult result = ObjectHandles.getGlobal().get(securityAnalysisResultHandle);
-            return createContingencyResultArrayPointer(result);
+            List<LimitViolationContext> limitViolations = result.getPreContingencyResult().getLimitViolationsResult().getLimitViolations()
+                .stream().map(limitViolation -> new LimitViolationContext("", limitViolation)).collect(Collectors.toList());
+            result.getPostContingencyResults()
+                .forEach(postContingencyResult -> limitViolations.addAll(postContingencyResult.getLimitViolationsResult()
+                    .getLimitViolations().stream()
+                    .map(limitViolation -> new LimitViolationContext(postContingencyResult.getContingency().getId(), limitViolation))
+                    .collect(Collectors.toList())));
+            return new SeriesPointerArrayBuilder<>(limitViolations)
+                    .addStringSeries("contingency_id", true, LimitViolationContext::getContingencyId)
+                    .addStringSeries("violation_id", true, LimitViolation::getSubjectId)
```

Comment:
we should name it "subject_id" like on java side, because it's not an identifier for the violation: we can have multiple violations for the same subject